web/php/utils/GlobalProperties.php

```diff
@@ -155,91 +155,91 @@ public static function init()
         self::$iniConfig = parse_ini_file(WIKIJUMP_ROOT . "/conf/wikijump.ini", true);
 
         // main settings
-        self::$WIKI_FARM                = self::fromIni("main", "wiki_farm", true);
+        self::$WIKI_FARM                = $_ENV["WIKI_FARM"] ?? self::fromIni("main", "wiki_farm", true);
```

Comment:
We should probably refactor this into a more generic configuration loader in the future, lol

Reply:
Yeah, agreed. The pattern will be essentially the same, I do want every facet here to be able to be overridden by environment variables but it could be presented in a nicer way.